### ISSUE TITLE
shadowDomCheck does not work in IE11

### ISSUE THREAD

**Issue by jasperbosch** [ionic-team/stencil#2151]
**Stencil version:**

```
 @stencil/core@1.8.5
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**

IE 11 tries to execute .includes('[native') on `undefined` wich results in an error.

**Expected behavior:**

IE 11 should not get an error.

**Steps to reproduce:**

Try to open an Vue.js application that uses stencil-components in IE11.

**Related code:**

In `src/client/client-windows.ts` line 20:
```tsx
export const supportsShadowDom = (BUILD.shadowDom) ? /*@__PURE__*/(() => (doc.head.attachShadow + '').includes('[native'))() : false;
```
however the `doc.head.attachShadow` is `undefined` in IE11

**Other information:**

Please revert [84b8e68](https://github.com/ionic-team/stencil/commit/8b5aa06e31919b16a9e11268441d250e254da36f#diff-01628de79373ea5a34a4e32838ac8603)

**Comment by adamdbradley** [comment]
Can you put together a repo case for this? Tests have been passing on ie11.

**Comment by jasperbosch** [comment]
Repo containing error: https://github.com/jasperbosch/jbo-vue-app
Repo containing component: https://github.com/jasperbosch/jbo-components

**Comment by adamdbradley** [comment]
Ok I see, thanks for the repo. In this case the app is not polyfilling `includes` before its first use. To keep it simple we can use `indexOf` to help out this scenario. We'll get it fixed in the 1.9.0 prerelease. Thanks
